src/main/java/practice/StreamPractice.java

```diff
@@ -1,20 +1,36 @@
 package practice;
 
-import java.util.Collections;
 import java.util.List;
+import java.util.NoSuchElementException;
+import java.util.Objects;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+import java.util.stream.Stream;
 import model.Candidate;
+import model.Cat;
 import model.Person;
 
 public class StreamPractice {
+    private static final CandidateValidator candidateValidator = new CandidateValidator();
+    private static final String SEPARATOR_FOR_NUMBERS_STRING = ",";
     /**
      * Given list of strings where each element contains 1+ numbers:
      * input = {"5,30,100", "0,22,7", ...}
      * return min integer value. One more thing - we're interested in even numbers.
      * If there is no needed data throw RuntimeException with message
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
+
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        return numbers
+                .stream()
+                .flatMap(s -> Stream.of(s.split(SEPARATOR_FOR_NUMBERS_STRING)))
+                .filter(s -> !Objects.equals(s, ""))
+                .mapToInt(Integer::valueOf)
```

Comment:
Filter is useless here 
better to use parseInt, cause it returns primitive when valueOf - return wrapper 
```suggestion
                .mapToInt(Integer::parseInt)
```